.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+*PR Title should start with one of the following tags: [Fix]/[Feature]/[Style]/[Update]*
+
```

Comment:
**I might state:**
Pull request Title must start with commit tag.  Below shows examples of when to use each tag:
[Fix]: A bug fix in the document​ 
[Feat]: New feature description added in document​ 
[Style]: Grammar or branding fix​ 
[Update]: Release update; updating Quartus versioning, tools, etc.

Reply:
this will help team to know when to use each

Reply:
I think this is self explanatory, as the primary descriptor is in the explanation, but I have added it to the PR template ensure no ambiguity with PRs.

Reply:
Should we add some commit types to directly address the commit history, like cherry-picking?

Reply:
Hi Tamim,  I know the team uses cherry-picking to create a release branch.  How would you see the customer using the info on what was cherry-picked?

Reply:
Cherry-picked commits will inherit the commit message it was cherry-picked from by default; I'd suggest we take advantage of that to follow the template guidelines.

I think it's useful, however, to notate in the cherry-picked commit message that it *is* a cherry pick from another branch. There's a number of ways to do it, but `git commit --amend` immediately after the [singular] `git cherry-pick` operation is probably the easiest for folks to remember.

Reply:
Okay -- then let's not complicate things further by adding any more commit "types".